ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -45,12 +44,24 @@
               "attributes": {
                 "memoryLimitBytes": "2147483648"
               }
+            },
+            "machine-exec": {
+              "servers": {
+                "machine-exec": {
+                  "attributes": {
+                    "type": "terminal"
+                  },
+                  "protocol": "ws",
+                  "port": "4444"
+                }
+              },
+              "installers": []
             }
           },
           "recipe": {
-            "contentType": "",
-            "type": "dockerimage",
-            "content": "eclipse/che-theia:0.3.14-nightly"
+            "contentType": "application/x-yaml",
+            "type": "compose",
+            "content": "services:\n  machine-exec:\n    image: 'aandrienko/che-machine-exec:latest'\n    mem_limit: 536870912\n  theia:\n    image: 'eclipse/che-theia:0.3.14-nightly'\n    mem_limit: 1073741824\n    depends_on:\n      - machine-exec\n"
```

Comment:
is that `image: 'aandrienko/che-machine-exec:latest` right image to have in master?

Reply:
Yes, exactly. CQ for kubernetes lib for che-machine-exec spends a lot of time, for now we can't merge it, so I used image from my fork.

Reply:
cc: @skabashnyuk

Reply:
ok